tck/src/main/java/org/eclipse/microprofile/concurrency/tck/MPConfigTest.java

```diff
@@ -406,6 +412,40 @@ public void overrideContextPropagationForThreadContextParameterWithConfig() {
         }
     }
 
+    /**
+     * Verify that MicroProfile config overrides the cleared, propagated, and unchanged attributes
+     * of a ThreadContext that is produced by the container when the application defines
+     * an unqualified parameter injection point that is annotated with ThreadContextConfig,
+     * where the ThreadContext parameter is the third parameter of the method.
+     */
+    @Test(dependsOnMethods = "beanInjected")
+    public void overrideContextPropagationForThreadContextParameter3WithConfig() {
+
+        // Expected config is propagated=Label; unchanged=ThreadPriority, cleared=Remaining
+        Assert.assertNotNull(producedThreadContext,
+                "Unable to inject ThreadContext into method parameter 3. Cannot run test.");
```

Comment:
Parameter's 1 & 2 might also cause problems, and this message could then be misleading.

Reply:
Added a commit which makes the failure message more generic to cover other possible causes.  Now it just says unable to inject.